basiclti/tsugi-util/src/java/org/tsugi/ags2/objects/Result.java

```diff
@@ -28,6 +28,9 @@
 	public static final String MIME_TYPE = "application/vnd.ims.lis.v2.result+json";
 	public static final String MIME_TYPE_CONTAINER = "application/vnd.ims.lis.v2.resultcontainer+json";
 
+	public static final String CONTENT_TYPE = MIME_TYPE + "; charset=utf-8";
```

Comment:
Is there a reason for having these same constants in `LineItem` and `Result`? **Looks** like `LTI13Servlet` is the only place these are used and they're just the same. Are these ever different? Seems like they could just be in `LTI13ConstantsUtil`?

Reply:
Matt are we both working on this?

Reply:
They are not the same. One is "lineitem" and the other is "result".  Where things belong in LTI depends on what is going on.   Everything in the basiclti folder that is "tsugi-util" is a literal translation of the IMS specs into generic (not Sakai specific) java code.  

LTI13Servlet is in basiclti-blis - which is Sakai specific.  It uses these spec defined constants.

The tsugi-util code is separately released in Sonatype as

<dependency>
   <groupId>org.tsugi</groupId>
   <artifactId>tsugi-util</artifactId>
   <version>0.4-SNAPSHOT</version>
</dependency>